vaadin-dialog-flow-parent/vaadin-dialog-flow/src/test/java/com/vaadin/flow/component/dialog/DialogTest.java

```diff
@@ -123,7 +115,7 @@ public void addDialogCloseActionListener_dialogClosed_JavaScriptIsScheduled() {
 
         dialog.open();
 
-        assertInvocations(8);
+        assertInvocations(9);
```

Comment:
The new `executeJs` call adds a new invocation so the number needs to be increased by one in the tests.